### PR TITLE
mlkube CI setup

### PR DESCRIPTION
**Pull request by foxish**
This is still WIP and sets up CI on GKE for https://github.com/foxish/mlkube.io (soon to move to a different org)

Todo:
- [x] Setup permissions to access cloud-sdk-testing bucket
- [x] Setup mount paths and test script

cc @jlewi

### CODE REVIEW

File: jobs/ci-kubernetes-e2e-mlkube-gke.env [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### job-env
+
+CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER=https://container.googleapis.com/
+PROJECT=mlkube-testing
+ZONE=us-central1-b
+KUBE_GKE_IMAGE_TYPE=gci
+E2E_TEST=false
```
Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-mlkube-gke.env]: use `--test=false` in config.json

File: jobs/config.json [hunk 2]
```diff
@@ -7587,5 +7587,22 @@
...
+    "args": [
+      "--env-file=jobs/platform/gke.env",
+      "--env-file=jobs/ci-kubernetes-e2e-mlkube-gke.env",
+      "--test=false",
+      "--charts",
+      "--mount-paths=$GOPATH/src/github.com/foxish/mlkube.io:/src/k8s.io/charts",
+      "--timeout=50m",
+      "--extract=gke",
+      "--check-leaked-resources",
+      "--mode=docker"
```
Comment (krzyzacy) [inline jobs/config.json]: remove this

File: jobs/config.json [hunk 3]
```diff
@@ -7587,5 +7587,22 @@
...
+    ],
+    "scenario": "kubernetes_e2e",
+    "sigOwners": [
+      "UNKNOWN"
```
Comment (krzyzacy) [inline jobs/config.json]: add a sig here if possible

File: prow/config.yaml [hunk 4]
```diff
@@ -13811,4 +13811,37 @@ periodics:
...
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      - name: USER
+        value: prow
+      - name: JENKINS_GCE_SSH_PRIVATE_KEY_FILE
+        value: /etc/ssh-key-secret/ssh-private
+      - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
+        value: /etc/ssh-key-secret/ssh-public
+      image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170606-e69a3df0
```
Comment (krzyzacy) [inline prow/config.yaml]: use kubekins-e2e-prow:v20170707-54693f85

**Comment by krzyzacy** [comment]
ah, I think the branch is not super up-to-date :-)

also you'll need to add a testgrid entry

**Comment by foxish** [comment]
@krzyzacy Fixed PTAL.

File: jobs/ci-kubernetes-e2e-mlkube-gke.env [hunk 5]
```diff
@@ -0,0 +1,6 @@
+### job-env
+
+CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER=https://container.googleapis.com/
+PROJECT=mlkube-testing
+ZONE=us-central1-b
+KUBE_GKE_IMAGE_TYPE=gci
```
Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-mlkube-gke.env]: minor nit: new line in the end

**Pull request merged**